### DISCUSSION TITLE
i am very angry

### DISCUSSION THREAD

**Discussion by smellysox1** [RasperRevision/RasperRevision.github.io#16] [General]
an hour of p2 was deleted bc the create thing  download button wouldn't work and there was nothing i could do. ;(

**Comment by smellysox1** [comment]
i will now on be a good boy and use json again, as i should

**Comment by smellysox1** [comment]
nvm spinning the whole site was the best failiure ive ever had.

**Comment by RasperRevision** [comment]
> an hour of p2 was deleted bc the create thing download button wouldn't work and there was nothing i could do. ;(

There is a bug where if you delete the last element you can't download the file. I'm sorry I didn't warn you about that or fix it.

**Reply by smellysox1** [reply]
ok. avoid that when demoing?

**Reply by RasperRevision** [reply]
yes

**Comment by smellysox1** [comment]
rafi movegrad looks ugly. i cannot into smooth animation. that is why it is left unused

**Comment by RasperRevision** [comment]
if we want the gradient to spin, we need to use a ::Before element. I don't think we have enough time to make that worth it.

**Reply by smellysox1** [reply]
i tried a before element yesterday and it didnt work

**Reply by smellysox1** [reply]
we should work on that later.

**Comment by smellysox1** [comment]
what should i do instead?
